src/sortStudents.ts

```diff
@@ -1,16 +1,60 @@
 
 export interface Student {
-  // describe Student interface
+  name: 'Jessica',
+  surname: 'Buxton',
+  age: 26,
+  married: true,
+  grades: [5, 5, 4, 5, 4, 4, 4, 4, 5, 4, 5, 4],
```

Comment:
Interface should contain types, not values
```suggestion
  name: string,
  surname: string,
  age: number,
  married: boolean,
  grades: number[],
```